batching/src/main/java/com/flipkart/batching/listener/NetworkPersistedBatchReadyListener.java

```diff
@@ -129,6 +129,7 @@ void unregisterReceiver() {
     void registerReceiverIfRequired() {
```

Comment:
Do we need the receiverRegistered any more? It should be based on the nullability of networkBroadcastReceiver